snowexsql/tables/measurement_type.py

```diff
@@ -15,17 +15,17 @@ class MeasurementType(Base):
     derived = Column(Boolean, default=False)
 
 
-
 class HasMeasurementType:
     """
     Class to extend when including a measurement type
     """
 
     @declared_attr
-    def measurement_type_id(cls):
-        return Column(Integer, ForeignKey('public.measurement_type.id'),
-                      index=True)
+    def measurement_type_id(self):
+        return Column(
+            Integer, ForeignKey('public.measurement_type.id'), index=True
+        )
 
     @declared_attr
-    def measurement(cls):
+    def measurement_type(self):
```

Comment:
This was a leftover from a PR that renamed all mentions of measurement to measurement_type